src/graphql/ingester/podDeployments.graphql

```diff
@@ -0,0 +1,9 @@
+type PodDeployment @rootType {
+    deploymentName: String! @compositeId
+    imageTag: String! @compositeId
+    targetReplicas: Int!
+    sha: String! @compositeId
+    previousSha: String!
+    environment: String!
+    timestamp: Int!
+}
```

Comment:
if we had constraints on this custom type, I'm assuming that `environment`, `sha`, `imageTag` must reference valid fields in other parts of the model.  Is that right?